### PR TITLE
Curtains now start anchored and can be moved with a screwdriver

### LINKED ISSUE #16778: Should be able to secure curtains.
It would be a nice feature if you could secure curtains in place. Currently they blow around due to ZAS and move on conveyor belts, which seems a little counter intuitive. Maybe just have the curtains blow open when there is ZAS around.

### PR DESCRIPTION
**Pull request by Sooose**
closes #16778
:cl:
 * tweak: Curtains now start anchored and can be (un)secured with a screwdriver.

### CODE REVIEW

**Comment by SonixApache** [comment]
unmovable curtains reduce meme potential

**Comment by Exxion** [comment]
*What* meme potential?

**Comment by Sooose** [comment]
Being able to move them on top of you to hide like a plant, I suppose. You can still wire cut them to get the full cost of making them back.

**Comment by DamianX** [comment]
If memes are a concern, a way to anchor/unanchor them could work.

File: code/game/objects/structures/curtains.dm [hunk 1]
```diff
@@ -36,11 +36,16 @@
 	if(iswirecutter(W))
 		playsound(loc, 'sound/items/Wirecutter.ogg', 50, 1)
 		if(do_after(user, src, 10))
-			to_chat(user, "<span class='notice'>You cut the shower curtains down.</span>")
+			to_chat(user, "<span class='notice'>You cut the [src] down.</span>")
```
Comment (DamianX) [inline code/game/objects/structures/curtains.dm]: This should be `\the [src]` or just `[src]`

File: code/game/objects/structures/curtains.dm [hunk 2]
```diff
@@ -36,11 +36,16 @@
...
 			var/obj/item/stack/sheet/mineral/plastic/A = getFromPool(/obj/item/stack/sheet/mineral/plastic, get_turf(src))
 			A.amount = 4
 			qdel(src)
 		return 1
+	if(isscrewdriver(W))
+		playsound(loc, 'sound/items/Screwdriver.ogg', 50, 1)
+		user.visible_message("[user] [anchored? "unsecures" : "secures"] the [src].", "You [anchored? "unsecure" : "secure"] the curtains.")
```
Comment (DamianX) [inline code/game/objects/structures/curtains.dm]: same here

**Review by DamianX — CHANGES_REQUESTED** [review]

**Comment by SonixApache** [comment]
with anchor options i re-declare my opinion on this a  :+1:

**Review by DamianX — APPROVED** [review]

**Pull request merged**